### PR TITLE
Theme setup

### PR DESCRIPTION
**Pull request by mbifulco**
Theme updates including header, footer, home, and cataog page

### CODE REVIEW

**Comment by tkeemon** [comment]
Great, thanks

File: templates/theme-footer.html [hunk 1]
```diff
@@ -5,22 +5,28 @@
...
       </nav>
-    </div>
-    <div class="references">
-      <span>Built on <a href="http://code.edx.org">OpenEdX</a>.</span>
+
+      <div class="built-on col-md-4 text-center">
+        <span>
+          Built on 
+          <a href="http://code.edx.org">
+            <img src="${static.url('images/edx-logo-bw.png')}" alt="Open EdX">
```
Comment (tkeemon) [inline templates/theme-footer.html]: This image isn't showing up under my devstack. You might want to try syntax closer to this:
https://github.com/appsembler/edx-theme/blob/appsembler/gymnasium/templates/theme-header.html#L8

**Comment by mbifulco** [comment]
Will update shortly and resubmit

**Comment by mbifulco** [comment]
@tkeemon give it a shot now

**Comment by tkeemon** [comment]
This looks really sharp @mbifulco . I especially like the layout of the course catalog page. 

Two minor things I noticed:
- The edx logo in the footer wasn't showing up. I left a comment inline with the code. Try changing the syntax to something closer to this:
  https://github.com/appsembler/edx-theme/blob/appsembler/gymnasium/templates/theme-header.html#L8
- On the course about page, overflow for `<div class="intro-inner-wrapper">` is hidden so I can't click on the button to enroll. 
  gymnasiumaboutheader

Other than that, everything looks awesome. Once those two things are fixed, I'll accept the PR and push it to the server.

**Comment by mbifulco** [comment]
@tkeemon both of those issues should be addressed.   Putting together a PR for platform shortly...

**Comment by tkeemon** [comment]
Looks great. I'm pushing to the server now.

**Pull request merged**